docs/5.0/enterprise/quickstart-enterprise.md

```diff
@@ -178,34 +178,50 @@ The output will look like this (re-formatted here to use compact YAML
 representation for brevity):
 
 ```yaml
+
 kind: role
-version: v3
 metadata:
   name: admin
 spec:
-  options:
-    cert_format: standard
-    forward_agent: true
-    max_session_ttl: 30h0m0s
-    port_forwarding: true
-  # allow rules:
   allow:
     logins:
     - '{% raw %}{{internal.logins}}{% endraw %}'
     - root
     node_labels:
       '*': '*'
+    app_labels:
+      '*': '*'
+    kubernetes_groups:
+    - '{% raw %}{{internal.kubernetes_groups}}{% endraw %}'
+    kubernetes_labels:
+      '*': '*'
+    kubernetes_users:
+    - '{% raw %}{{internal.kubernetes_users}}{% endraw %}'
     rules:
     - resources: [role]
       verbs: [list, create, read, update, delete]
     - resources: [auth_connector]
       verbs: [list, create, read, update, delete]
+    - resources: [user]
+      verbs: [list, create, read, update, delete]
     - resources: [session]
       verbs: [list, read]
+    - resources: [event]
+      verbs: [list, read]
     - resources: [trusted_cluster]
       verbs: [list, create, read, update, delete]
-  # no deny rules are present, the admin role must have access to everything)
+    - resources: [token]
+      verbs: [list, create, read, update, delete]
   deny: {}
+  options:
+    cert_format: standard
+    enhanced_recording:
+    - command
+    - network
```

Comment:
Take this extra stuff out, I know Teleport adds it back in but we should give examples which are as simple as possible.

```suggestion
```